src/lua/api-gateway/validation/factory.lua

```diff
@@ -101,9 +102,9 @@ local function _generateHmacSignature()
     return hmacSignatureValidator:generateSignature()
 end
 
-local function _validateOAuthToken()
+local function _validateOAuthToken(obj)
```

Comment:
Thanks @constantincristian . I think it would be helpful to add a test with this as well.